src/main/webapp/app/shared/routes/route-selector.component.scss

```diff
@@ -9,27 +9,30 @@ Route selector styles
     input[type="search"] {
         margin: auto 10px;
     }
-    .dropdown-menu {
-        border-color: darkgrey;
-        .dropdown-item { /* routes names */
-            margin-right: 0.5rem;
-            word-break: break-all;
-            display: block;
-            display: -webkit-flex;
-            justify-content: space-between;
-            &:hover {
-                color: #fff;
-                background-color: #80bed8;
-            }
-            &:active {
-                color: #fff;
-                background-color: #0275d8;
-            }
-            .badge { /** vertically center badges */
-                margin: auto 0;
-            }
-            &.disabled { /** disable DOWN instances **/
-                pointer-events: none;
+    div {
```

Comment:
dont use element selector as it might leak into all elements and its expensive. Use a class selector and apply to elements having only that class